customScripts/expansiontalk.gd

```diff
@@ -23,8 +23,11 @@ var person
 ###Randomly choose the name for "Master" each Convo
 func getMasterName(person):
 	var names = []
-	var related = globals.expansion.relatedCheck(globals.player, person)
+	var related = ''
```

Comment:
This function is reverted back to a bugged state.

Reply:
fixed for next commit